tests/unit-tests/platforms/gbm-kms/kms/test_display_multi_monitor.cpp

```diff
@@ -147,12 +149,17 @@ class MesaDisplayMultiMonitorTest : public ::testing::Test
 
     std::shared_ptr<mgg::Platform> create_platform()
     {
+        mir::udev::Context ctx;
+        // Caution: non-local state!
+        // This works because standard-drm-devices contains a udev device with 226:0 and devnode /dev/dri/card0
+        auto device = ctx.char_device_from_devnum(makedev(226, 0));
+       
```

Comment:
I see this setup repeated multiple times, worth factoring out?

Reply:
There's no obvious place for it, but yeah, it would be worth factoring out.